pkgs/games/steam/fhsenv.nix

```diff
@@ -1,46 +1,37 @@
-{ config, lib, writeScript, buildFHSUserEnv, steam, glxinfo-i686
+{ lib, stdenv, writeScript, buildFHSUserEnv, steam, glxinfo-i686, runtimeShell
 , steam-runtime-wrapped, steam-runtime-wrapped-i686 ? null
 , extraPkgs ? pkgs: [ ] # extra packages to add to targetPkgs
 , extraLibraries ? pkgs: [ ] # extra packages to add to multiPkgs
 , extraProfile ? "" # string to append to profile
 , extraArgs ? "" # arguments to always pass to steam
-, runtimeOnly ? false
-, runtimeShell
-, stdenv
-
-# DEPRECATED
-, withJava ? config.steam.java or false
-, withPrimus ? config.steam.primus or false
+, noGameSpecificLibraries ? false # exclude game specific libraries
```

Comment:
Maybe rename this to `withGameSpecificLibraries` or something like that to avoid the awkward "not noGameSpecificLibraries" check later?